Tumusime14/Certificate-Manager---David
# Feature/Task-2-KAN-45 Adding table to Example 1

## PR Description

### Feature: 

- Display Table on menu-item click (Example 1)

This pull request introduces a new feature that displays a table when clicking on the menu item "Example 1".
This feature enhances the user interface by providing an organized way to present data in a tabular format.

### Ticket / Jira Link:
[Feature/Task-2-KAN-45](https://task-force-pro-dccs.atlassian.net/jira/software/projects/KAN/boards/1?assignee=712020%3Ab9bb79c7-4352-4d17-9af2-5c36278b1670%2C712020%3A5d939816-4bad-4fed-be90-a36cfb42d87f&selectedIssue=KAN-45)

### Checklist
- [x]  The feature behaves as described in the requirements.
- [x]  The feature does not generate warnings or errors.

### How To Test Changes:

1. Pull the latest changes from the feature branch
     `git pull origin Feature/Task-2-KAN-Adding-table`
2. Start the development server
     `npm start`
3. Navigate to the application in your web browser.
4. Click on the menu item "Example 1" and verify that a table appears, 
displaying the data with the specified columns.

### Additional Comments

This feature provides a dynamic way to view detailed information by displaying data in a structured table format.
The implementation leverages React components and CSS to ensure the table is presented as specified.

### Screenshot
![image](https://github.com/user-attachments/assets/c24854be-cf72-4b63-836c-5ec484b7060a)
